### ISSUE TITLE
Hiding "About" link will result in plain Steam page

### ISSUE THREAD

**Issue by MxtOUT** [tfedor/Enhanced_Steam#44]
When I'm enabling the "Hide 'About' link" feature in version 0.9.2 on Chrome 72.0.3626.96 every other feature stops working and the store pages will look like without the extension

**Comment by KarlCastle** [comment]
Hmm. I've tried replicating this, and I can't either logged in or logged out. So I need some more information.

Is the storefront in a language other than US English?

Do you get any warnings or errors in the browser console? (F12) Usually "extension stops working entirely" is due to a javascript error.

**Comment by MxtOUT** [comment]
The storefront is in AE.
I got the following error in the console:
```
common.js:1327 Uncaught (in promise) TypeError: Cannot read property 'remove' of null
    at Object.self.removeAboutMenu (common.js:1327)
    at Object.self.init (common.js:2637)
    at store.js:2917
```
which should be fixed with #50.

On my client the "About" link is replaced with a "Chat" link when I log in:

Logged out:
![about_link](https://user-images.githubusercontent.com/14999077/52799526-48fa0900-307a-11e9-9ae2-082fe7582b7a.png)

Logged in:
![chat_link](https://user-images.githubusercontent.com/14999077/52799541-4eefea00-307a-11e9-9695-20d419bb6c4c.png)
